vignettes/estimate_infections.Rmd

```diff
@@ -33,31 +33,35 @@ These infections are then mapped to observations via discrete convolutions with
 
 ### Initialisation
 
-The model is initialised before the first observed data point by assuming constant exponential growth for the mean of modelled delays from infection to case report (called `seeding_time` $t_\mathrm{seed}$ in the model):
+The model is initialised before the first initerved data point by assuming constant exponential growth for the mean of modelled delays from infection to case report (called `seeding_time` $t_\mathrm{seed}$ in the model):
```

Comment:
```suggestion
The model is initialised before the first data point by assuming constant exponential growth for the mean of modelled delays from infection to case report (called `seeding_time` $t_\mathrm{seed}$ in the model):
```

 I was a bit confused by this so just removed.

Reply:
failed search replace "obs" -> "init" (which I thought was a better descriptive index for I/r to match the code).